include/sys/zfs_ioctl.h

```diff
@@ -636,7 +648,7 @@ extern void zfs_unmount_snap(const char *);
 extern void zfs_destroy_unmount_origin(const char *);
 extern int getzfsvfs_impl(struct objset *, struct zfsvfs **);
 extern int getzfsvfs(const char *, struct zfsvfs **);
-extern void latency_stats(uint64_t* histo, unsigned int buckets,
+extern void latency_stats(uint64_t * histo, unsigned int buckets,
     stat_pair* lat);
 
```

Comment:
additional space is there after * in both lines

Reply:
done